facdb_build/sql/dcp_colp.sql

```diff
@@ -32,10 +33,10 @@ ALTER TABLE dcas_colp
 	ADD	captype text,
 	ADD	proptype text;
 
-UPDATE dcas_colp
-SET usedec = REPLACE(usedec, 'NO USE-NON RESIDENTIAL STRUCTURESE', 'NO USE-NON RESIDENTIAL STRUCTURE'); 
+UPDATE dcp_colp
+SET usetype = REPLACE(usetype, 'NO USE-NON RESIDENTIAL STRUCTURESE', 'NO USE-NON RESIDENTIAL STRUCTURE'); 
```

Comment:
Open an issue in COLP to make this an enhancement.  Optional to leave the change here

Reply:
I opened an issue. I think the correction should stay in the FacDB workflow for now since we're pulling COLP from Bytes, rather than our own database.